### DISCUSSION TITLE
Stenographer diskfreepercentage issue

### DISCUSSION THREAD

**Discussion by antakeo** [Security-Onion-Solutions/securityonion#8390] [General]
Hello,
Having issues with stenographer. I followed this guide to adjust diskfreepercentage to 25, so Im assuming that when /nsm gets past 75% usage (df -h /nsm) then pcaps would start to purge. However /nsm continues to reach past 90%. I can cron job so-pcap-restart, but I think that defeats the purpose.
So version is 2.3.91. Am I missing another config somewhere, is my stenographer broken?

**Comment by dougburks** [comment]
> I followed this guide to adjust diskfreepercentage to 25

You didn't say which guide you followed. I assume it was this one:
https://docs.securityonion.net/en/2.3/stenographer.html#configuration

You might want to double-check your syntax and spacing.

> So version is 2.3.91

You probably want to update to the latest version as much has changed since 2.3.91 was released in 2021:
https://docs.securityonion.net/en/2.3/soup.html
